lms/static/js/spec/instructor_dashboard/instructor_dashboard_spec.js

```diff
@@ -0,0 +1,22 @@
+define(['jquery', 'coffee/src/instructor_dashboard/instructor_dashboard'],
+    function($) {
+        'use strict';
+        
+        describe("LMS Instructor Dashboard", function() {
+            
+            beforeEach(function() {
+                loadFixtures('js/fixtures/instructor_dashboard/instructor_dashboard.html');
```

Comment:
Because this test is simply checking the structure of a fixture file, it doesn't have any value. There is no way to enforce that the fixture file is the same as the HTML actually returned on the server. I think you should delete it and the fixture file.

Fixture files should only be used for testing JavaScript interaction (in which case you do need the HTML elements present, and the actual structure of the HTML is not that important).